src/v8/inspector/messages.rs

```diff
@@ -0,0 +1,279 @@
+/*
+ * Copyright Redis Ltd. 2022 - present
+ * Licensed under your choice of the Redis Source Available License 2.0 (RSALv2) or
+ * the Server Side Public License v1 (SSPLv1).
+ */
+//! The messages the V8 Inspector, represented by [super::Inspector],
+//! sends and receives.
+//!
+//! See [ClientMessage], [MethodInvocation], [ServerMessage],
+//! and [ErrorMessage].
+//!
+//! For more information on the protocol, see the official V8
+//! documentation:
+//! <https://chromedevtools.github.io/devtools-protocol/v8/>.
+use serde::{Deserialize, Serialize};
+
+use serde_aux::prelude::*;
+
+/// The error subset of dispatch codes of the v8 inspector protocol. A
+/// copy from the "dispatch.h" header file.
+#[derive(Debug, Copy, Clone)]
+pub enum ErrorCode {
```

Comment:
Can this be auto generated somehow? Otherwise we will need to maintain it each time it changes, right?